Loops-Arrays/loops-arrays-task.cs

```diff
@@ -0,0 +1,241 @@
+    int sum = 0;
+    for (int i = 0; i <= positiveNumber; i++)
+    {
+        sum += i;
+    }
+    Console.WriteLine($"The sum of numbers from 0 to the given number : {sum}");
+}
+//Task 2
+//Using a while loop write the multiplication table for the number 3 to the console.
+{
+    Console.WriteLine("Task 2. Multiplication table for the number 3: ");
+    int number = 3;
+    int multiplication = 0;
+    while (multiplication <= 10)
+    {
+        Console.WriteLine($"{number} * {multiplication} = {number * multiplication}");
+        multiplication++;
+    }
+}
+
+//Task 3
+//Create an array of numbers [3, 5, 9, 8, 15]. Find the product of these numbers and write the result to the console.
+{
+    int[] numbers = { 3, 5, 9, 8, 15 };
+    int result = 1;
+
+    foreach (int value in numbers)
+    {
+        result *= value;
+    }
+    Console.WriteLine($"Task 3. Product of array [3, 5, 9, 8, 15] : {result}");
+}
+//Task 4
+//Write a program that calculates how many times 2048 must be divided by 2 to make it less than 10.
+{
+    int calk = 0;
+    for (int numberToDevide = 2048; numberToDevide > 10; calk++)
+    {
+        numberToDevide /= 2;
+    }
+    Console.WriteLine($"Task 4. How many times 2048 must be divided by 2 to make it less than 10 : {calk} \n");
+}
+//Task 5
+//Create an array of strings. Write a program that analyzes a created array and, if there is a string with the value “Hello”,
+//displays the word “Labas!” in console and leaves the loop once it is found.
+{
+    Console.Write("Task 5.");
+    string[] greetings = { "Chao", "Hola", "Hello", "Cześć" };
+    for (int i = 0; i < greetings.Length; i++)
+    {
+        if (greetings[i] == "Hello")
+        {
+            Console.WriteLine("Labas!");
+            break;
+        }
+    }
+}
+//Task 6
+//Create an array of numbers. Write a program that calculates the sum of the first and last element of an array and writes it to the console.
+{
+    int[] numbersArray = { 3, 5, 9, 8, 15 };
+    int sum = numbersArray[0] + numbersArray[numbersArray.Length - 1];
+    //int sum = 0; // solution with cycles
+    //for (int i = 0; i < numbersArray.Length; m++)
+    //{
+    //    if (i == 0 || i == (numbersArray.Length - 1))
+    //    {
+    //        sum += numbersArray[i];
+    //    }
+    //};
+    Console.WriteLine($"Task 6. Sum of the first and last element of an array: {sum}");
+}
+//Task 7
+//Create an array of numbers. Find the sum of indexes of minimum and maximum array elements.
+{
+    int[] numbersArray = { 3, 5, 1, 8, 15 };
+    int min = numbersArray[0];
+    int max = numbersArray[0];
+    for (int i = 0; i < numbersArray.Length; i++)
+    {
+        if (numbersArray[i] > max)
+        {
+            max = numbersArray[i];
+        }
+
+        if (numbersArray[i] < min)
+        {
+            min = numbersArray[i];
+        }
+
+    }
+    Console.WriteLine($"Task 7.Max number is {max} and Min number is {min}");
```

Comment:
final result should display sum of indexes of max and min elements. Please rewrite it according to the conditions